src/middlewares/userMiddlewares.js

```diff
@@ -18,11 +19,30 @@ export default class UserMiddlewares {
   static async checkUserExists(req, res, next) {
     try {
       const { email } = req.body;
-      const data = await userServices.getUserByEmail(email);
+      const data = await UserServices.getUserByEmail(email);
       if (!data) return next();
       return resError(res, 409, 'Unsuccesful, user already exists, kindly use a different email.');
     } catch (error) {
       return resError(res, 500, error.message);
     }
   }
+
+  /**
+ * @name userVerified
+ * @description Checks if a user exists in the database
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @param {object} next The response object
+ * @returns {object} The API response or next()
+ */
+  static async isUserVerified(req, res, next) {
+    try {
+      const userDetails = Utils.verifyToken(req.query.token);
```

Comment:
Please, confirm from Anna and  get back to me, I think token should be gotten from the headers and not query param.

Reply:
@okobaba1 How do you suggest @misanj use headers to get the token from another route? I think she will be able to only get the token from headers if it was from the client-side.

Reply:
> Please, confirm from Anna and get back to me, I think token should be gotten from the headers and not query param.

@okobaba1 How do you suggest @misanj use headers to get the token from another route? I think she will be able to only get the token from headers if it was from the client-side.